react/components/LogoutCustomerSession.tsx

```diff
@@ -49,6 +53,24 @@ interface Props {
 const LogoutCustomerSession = (props: Props) => {
   const { client, loading, onDepersonify, attendantEmail, mobile } = props
   const handles = useCssHandles(CSS_HANDLES)
+  const { orderForm } = useOrderForm();
```

Comment:
Lint this file.

Reply:
@igorbrasileiro I tried few things to do linting but I couldn't. Can you give any command to do lint ?

Reply:
Oh, I believe the problem is in the command specified in `react/package.json`: `"lint": "tsc --noEmit --pretty && tslint -c tslint.json './**/*.ts'"` I think it doesn't look for the `.tsx` files.

You can run `tslint -c tslint.json components/LogoutCustomerSession.tsx --fix` inside the `/react` folder to fix what @igorbrasileiro asked.

Reply:
@lariciamota I have used this command inside react folder but this is not working

![lint-error](https://user-images.githubusercontent.com/76150710/162670373-2e9c0e25-7add-4c2f-bb37-2dc251291b0b.PNG)
g

Reply:
@lariciamota Done the linting & made the requested changes

Reply:
@igorbrasileiro I tried the command which you shared. It was not working, but I was able to do linting by using the command which is shared by @lariciamota